indico/cli/i18n.py

```diff
@@ -335,6 +335,68 @@ def remove_empty_pot_files(path: Path, python=False, javascript=False, react=Fal
         _get_messages_react_pot(path).unlink()
 
 
+# Merge after generating all the .pot files
+def merge_pot_files(output_file, *input_files):
+    """Merge multiple POT files into a single POT file."""
+    merged_catalog = None
+
+    print('\n🗨  Merging POT files...\n')
+
+    for input_file in input_files:
+        with input_file.open('rb') as f:
+            catalog = read_po(f)
+
+            if merged_catalog is None:
+                merged_catalog = catalog
+            else:
+                for message in catalog:
+                    if message.id not in merged_catalog:
+                        merged_catalog[message.id] = message
+
+    if merged_catalog:
+        with output_file.open('wb') as f:
+            write_po(f, merged_catalog)
```

Comment:
This should never happen and if it does, the command should just fail, so I'd put an `assert` here instead:

```suggestion
    assert merged_catalog is not None
    with output_file.open('wb') as f:
        write_po(f, merged_catalog)
```